bin/configs/kotlin-misk-config.yaml

```diff
@@ -0,0 +1,10 @@
+generatorName: kotlin-misk
+outputDir: samples/server/petstore/kotlin-misk-config
```

Comment:
what about adding the new folder to the CI workflow as well?

https://github.com/OpenAPITools/openapi-generator/blob/master/.github/workflows/samples-kotlin-server.yaml#50

Reply:
restarted and all passed